### PR TITLE
Update minikube and serialize tests to stabilize them

### PR DESCRIPTION
**Pull request by andresmgot**

### CODE REVIEW

File: scripts/install-minikube.sh [hunk 1]
```diff
@@ -9,39 +24,25 @@ touch ~/.kube/config
 
 export KUBECONFIG=$HOME/.kube/config
 
-MINIKUBE_VERSION=v0.21.0
+MINIKUBE_VERSION=v0.24.1
 
 install_bin() {
     local exe=${1:?}
-    test -n "${TRAVIS}" && sudo install -v ${exe} /usr/local/bin || install ${exe} /usr/local/bin
+    test -n "${TRAVIS}" && sudo install -v ${exe} /usr/local/bin || install ${exe} /usr/local/binq
```
Comment (sebgoa) [inline scripts/install-minikube.sh]: binq ? never seen that before

Comment (andresmgot) [reply scripts/install-minikube.sh]: true, it is a typo

**Comment by andresmgot** [comment]
merged changes in #104 to check the tests

**Pull request merged**